sql/telemetry_derived/experiment_search_aggregates_base/view.sql

```diff
@@ -0,0 +1,61 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry_derived.experiment_search_aggregates_live`
+AS
+WITH live_and_stable AS (
+  SELECT
+    *,
+    'telemetry_stable' AS dataset_id
+  FROM
+    `moz-fx-data-shared-prod.telemetry_stable.main_v4`
+  UNION ALL
+  SELECT
+    *,
+    'telemetry_live' AS dataset_id
+  FROM
+    `moz-fx-data-shared-prod.telemetry_live.main_v4`
+)
+SELECT
+  date(submission_timestamp) AS submission_date,
+  dataset_id,
```

Comment:
this view exposes `submission_date` and `dataset_id` so that it can be reused in both `experiment_search_aggregates_v1` and `experiment_search_aggregates_live_v1`